scanpipe/pipes/d2d.py

```diff
@@ -1283,3 +1283,25 @@ def scan_unmapped_to_files(project, logger=None):
     project.codebaseresources.files().to_codebase().filter(status=flag.SCANNED).update(
         status=flag.REQUIRES_REVIEW
     )
+
+
+def flag_deployed_resources_with_missing_license(project, doc_extensions=None):
+    """Update the status for deployed files with missing license."""
```

Comment:
The docstring and function name do not reflect properly what I'm seeing in the function code.
"for deployed files" -> It's a bit confusing since the QS works on "from/" files.

Reply:
These are from files that are deployed on `to` side.
I guess `flag_deployed_from_resources_with_missing_license` will be more appropriate function name.

Reply:
This is not explicit looking at the code, a comment about it would help the readability ;)

Reply:
ack